### PR TITLE
Fix development app

### PR DESCRIPTION
**Pull request by shellderp**
By installing MiskWebModule

### CODE REVIEW

File: service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt [hunk 1]
```diff
@@ -36,6 +37,7 @@ class BackfilaServiceModule(
     install(BackfilaWebActionsModule())
     install(BackfilaDashboardModule(environment))
     install(ServiceWebActionsModule())
+    install(AdminDashboardModule(environment))
```
Comment (shellderp) [inline service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt]: confused why this wasnt needed before.. is this installed in CashBackfilaService?

Comment (adrw) [reply service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt]: It's installed by Skim

Comment (shellderp) [reply service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt]: is it going to be broken if it's double installed?

Comment (shellderp) [reply service/src/main/kotlin/app/cash/backfila/service/BackfilaServiceModule.kt]: will merge, we can figure it out if it does break

**Pull request merged**